src/mobu/models/ci_manager.py

```diff
@@ -0,0 +1,47 @@
+"""Models for CiManager."""
+
+from pydantic import BaseModel, Field, HttpUrl
+
+from .user import User
+
+__all__ = [
+    "CiJobSummary",
+    "CiManagerSummary",
+    "CiWorkerSummary",
+]
+
+
+class CiJobSummary(BaseModel):
+    """Information about a job."""
+
+    commit_url: HttpUrl = Field(
+        ...,
+        title="GitHub URL to the commit being worked on",
```

Comment:
Here and throughout this file, `title` should be as short as possible, usually just a few words, and the long human-readable information should go into `description` instead. Also, adding `examples` to each field produces much nicer API documentation, since all those examples will turn into a sample reply from the API as JSON with real-looking data.